### PR TITLE
Contributing Guidelines and Code of Conduct

### PR DESCRIPTION
**Pull request by dondish**
Files that help future-contributors see how they can contribute.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -17,9 +17,21 @@ Please consider these following guidelines when contributing:
 * For any issue regarding memory leaks / high cpu usage please send a [diagnosis](https://nodejs.org/en/docs/guides/simple-profiling/) of your performance 
 * For any uncertainty of how to reproduce the issue, specify that as well! Describe your actions before the bug occured.
 
+### Submitting Enhancements
+We appreciate new ideas and improvements and we want Nino to reach its full potential! 
+If you have any enhancement you want done in Nino please submit an issue!
```
Comment (ohlookitsderpy) [inline CONTRIBUTING.md]: Stop shouting! We only need an exclamation mark once here

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -17,9 +17,21 @@ Please consider these following guidelines when contributing:
...
+
+Before submitting the issue, consider the following guidelines:
+* Be clear and concise with the title, it will help others link their issues and solutions to yours.
+* In the enhancement's description, please add a way to implement the idea. 
+  It doesn't have to be in code but rather an idea to make sure it's up to standards
```
Comment (ohlookitsderpy) [inline CONTRIBUTING.md]: Punctuation

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -17,9 +17,21 @@ Please consider these following guidelines when contributing:
...
+* Add examples of how the feature can be interacted with, it will make the general concept much more easier to grasp.
+
 ### Pull Requests
 Before submitting a PR please make sure there is no other similar PR submitted.
```
Comment (ohlookitsderpy) [inline CONTRIBUTING.md]: there is -> there are

Comment (ohlookitsderpy) [reply CONTRIBUTING.md]: PR should also be PRs

Comment (dondish) [reply CONTRIBUTING.md]: thanks!

File: CONTRIBUTING.md (hunk 1 above)
Comment (dondish) [reply CONTRIBUTING.md]: Thanks!

**Pull request merged**